### ISSUE TITLE
Making the collection working with community.kubernetes 2.0.0

### ISSUE THREAD

**Issue by Andersson007** [ansible-collections/community.kubevirt#33]
##### SUMMARY
Relates to https://github.com/ansible-collections/community.kubevirt/pull/25.

The collection has a dependency - the [community.kubernetes](https://github.com/ansible-collections/community.kubernetes) collection.
There were breaking changes introduced in `community.kubernetes 2.0.0` and `community.kubevirt` cannot work with it.

Currently the community.kubernetes versions are restricted in [galaxy.yml](https://github.com/ansible-collections/community.kubevirt/blob/main/galaxy.yml):
```
dependencies:
  community.kubernetes: '>=1.0.0,<2.0.0'
```

1) The issue is to make the collection working with community.kubernetes 2.0.0+

This will be a breaking change also, so will require the next major version and prior announcement some time before the release.

**OR**
2) Evaluate if the dependency is really needed? Maybe it's sensible to get rid of it? (I don't use the collections, so hard to say)

##### ISSUE TYPE
- Bug Report

**Comment by snecklifter** [comment]
@Andersson007 community.kubernetes has been renamed to kubernetes.core

I'm working on compatibility with this now. It does rely quite heavily on the kubernetes module so I think removing this dependency would involve a lot of re-inventing the wheel. Hopefully all the name changing and strange versioning is now behind us.

**Comment by Andersson007** [comment]
@snecklifter yeah, forgot about the renaming, thanks:) and yeah, if it relies on the dependency heavily, keeping it sounds sensible.
Thanks for working on this!
